airflow/www/views.py

```diff
@@ -336,11 +336,19 @@ def get_int_arg(value, default=0):
         state_color_mapping = State.state_color.copy()
         state_color_mapping["null"] = state_color_mapping.pop(None)
 
+        page_title = "Airflow - DAGs"
```

Comment:
why not add this to the constants in the module level scope?

Reply:
Yes, could do. I think it might help with readability if perhaps this is defined close to the logic.

Reply:
If it's only used in one place, I agree, keep our close to it's usage